assets/src/blocks/HubspotForm/HubspotFormBlock.js

```diff
@@ -0,0 +1,73 @@
+import { HubspotFormEditor } from './HubspotFormEditor';
+import { Tooltip } from '@wordpress/components';
+import { frontendRendered } from '../frontendRendered';
+
+const { __ } = wp.i18n;
+
+const BLOCK_NAME = 'planet4-blocks/hubspot-forms';
+
+const IMAGE_FULL_WIDTH = 'imageFullWidth';
+
+const getStyleLabel = (label, help) => {
+  if (help) {
+    return (
+      <Tooltip text={help}>
+        <span>{label}</span>
+      </Tooltip>
+    );
+  }
+  return label;
+};
+
+export class HubspotFormBlock {
```

Comment:
We don't use class components in new code any more. Can you change this to a function component?